pubspec.yaml

```diff
@@ -12,11 +12,11 @@ dependencies:
   logging: '>=0.9.0 <0.12.0'
   package_config: '>=0.1.5 <2.0.0'
   path: '>=0.9.0 <2.0.0'
-  stack_trace: ^1.3.0
-  vm_service: ^1.0.0
+  stack_trace: '^1.3.0'
```

Comment:
Remove changes that just add gratuitous quotes.

Reply:
Sure, I intended it to keep it consistent with the rest of the file, but I'll remove the gratuitous quotes